packages/contracts-bedrock/contracts/L2/L1Block.sol

```diff
@@ -2,6 +2,10 @@
 pragma solidity 0.8.15;
 
 import { Semver } from "../universal/Semver.sol";
+import { RingLib, RING_SIZE } from "../libraries/RingLib.sol";
+
+/// @notice Error code l1 block hash number out of bounds error.
+error L1BlockHashNumberOutOfBounds();
```

Comment:
We are planning on updating to use custom errors in the future, do you mind using a require in the meantime for standardizing things?